netbox_librenms_plugin/templates/netbox_librenms_plugin/_interface_sync.html

```
@@ -7,19 +7,21 @@ <h2>Interface Sync</h2>
     <div class="btn-list">
         <form method="post">
             {% csrf_token %}
+            {# Carry the active server_key so the refresh hits the right LibreNMS server/cache (else a non-default server tab uses the fallback). #}
+            {% if server_key %}<input type="hidden" name="server_key" value="{{ server_key }}">{% endif %}
             {% if has_librenms_id %}
                 {% with model_name=object|meta:"model_name" %}
                     {% if model_name == "device" %}
                         <button hx-post="{% url 'plugins:netbox_librenms_plugin:device_interface_sync' pk=object.pk %}"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)